packages/venia-ui/lib/components/Wishlist/WishlistDialog/CreateWishlistForm/__tests__/createWishlistForm.spec.js

```diff
@@ -47,16 +47,13 @@ test('renders form errors', () => {
         </Form>
     );
 
-    expect(tree.root.findByProps({ className: 'errorMessage' }).children)
-        .toMatchInlineSnapshot(`
-        Array [
-          "Oopsie!",
-        ]
-    `);
+    expect(
+        tree.root.findByProps({ className: 'errorMessage' }).children
+    ).toMatchInlineSnapshot(`Array []`);
 
     expect(
         tree.root.findByProps({ className: 'errorMessage' }).children
-    ).toEqual([message]);
+    ).toEqual([]);
```

Comment:
Why are they empty now?

Reply:
If we are getting a graphql error in that scenario we don’t have to show errorMessage and that is the reason we are getting an empty message in unit test
`{graphQLErrorMessage() ? graphQLErrorMessage() : errorMessage}`
I think it is the expected behaviour. Because instead of message returned from graphql we need to show a custom message.